tools_for_todoist/models/google_calendar.py

```diff
@@ -99,6 +99,7 @@ def _process_sync(self):
                     "Skipping recurring event exception for missing event: ", event)
                 continue
             self._events[recurring_event_id].update_exception(event)
+            updated_events.append(self._events[recurring_event_id])
```

Comment:
let's keep the original exception as the updated value. It can be handled downstream that this affects the entire recurring event